mapbox/services/geocoding.py

```diff
@@ -58,6 +58,12 @@ def forward(self, address, types=None, lon=None, lat=None, country=None):
             params.update(proximity='{0},{1}'.format(
                 round(float(lon), self.precision.get('proximity', 3)),
                 round(float(lat), self.precision.get('proximity', 3))))
+        if bbox is not None:
+            params.update(bbox='{0},{1},{2},{3}'.format(
+                bbox[0],
+                bbox[1],
+                bbox[2],
+                bbox[3]))
```

Comment:
When calling a function, you can use the `*some_list` syntax to "unpack" each element as a positional argument. So this could be expressed as:

```
params.update(bbox='{0},{1},{2},{3}'.format(*bbox)
```